cmake/Modules/FindCapstone.cmake

```diff
@@ -0,0 +1,29 @@
+if (UNIX)
+  find_package(PkgConfig REQUIRED)
+
+  pkg_check_modules(CAPSTONE REQUIRED capstone>=3.0.4)
+  include_directories(${CAPSTONE_INCLUDE_DIRS})
```

Comment:
Are this line and the next one actually needed? It seems they are present in the calling `CMakeLists.txt` anyway (after `find_package(Capstone REQUIRED)`), regardless of the platform.

Reply:
No, you're perfectly right, they don't belong there. Already fixed, thank you!